src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs

```diff
@@ -719,141 +719,18 @@ public static class Popcnt
-        public static Vector128<U> StaticCast<T, U>(Vector128<T> value) where T : struct where U : struct { throw null; }
-        public static Vector128<float> Shuffle(Vector128<float> left, Vector128<float> right, byte control) { throw null; }
-        public static Vector128<float> Sqrt(Vector128<float> value) { throw null; }
-        public static Vector128<float> SqrtScalar(Vector128<float> value) { throw null; }
-        public static Vector128<float> SqrtScalar(Vector128<float> upper, Vector128<float> value) { throw null; }
-        public static unsafe void StoreAligned(float* address, Vector128<float> source) { throw null; }
-        public static unsafe void StoreAlignedNonTemporal(float* address, Vector128<float> source) { throw null; }
-        public static unsafe void Store(float* address, Vector128<float> source) { throw null; }
-        public static void StoreFence() { throw null; }
-        public static unsafe void StoreHigh(float* address, Vector128<float> source) { throw null; }
-        public static unsafe void StoreLow(float* address, Vector128<float> source) { throw null; } 
-        public static unsafe void StoreScalar(float* address, Vector128<float> source) { throw null; }
-        public static Vector128<float> Subtract(Vector128<float> left, Vector128<float> right) { throw null; }
-        public static Vector128<float> SubtractScalar(Vector128<float> left, Vector128<float> right) { throw null; }
-        public static Vector128<float> UnpackHigh(Vector128<float> left, Vector128<float> right) { throw null; }
-        public static Vector128<float> UnpackLow(Vector128<float> left, Vector128<float> right) { throw null; }
-        public static Vector128<float> Xor(Vector128<float> left, Vector128<float> right) { throw null; }
-    }
     public static class Sse2
     {
         public static bool IsSupported { get { return false; } }
-        public static Vector128<byte> Add(Vector128<byte> left, Vector128<byte> right) { throw null; }
-        public static Vector128<sbyte> Add(Vector128<sbyte> left, Vector128<sbyte> right) { throw null; }
-        public static Vector128<short> Add(Vector128<short> left, Vector128<short> right) { throw null; }
-        public static Vector128<ushort> Add(Vector128<ushort> left, Vector128<ushort> right) { throw null; }
-        public static Vector128<int> Add(Vector128<int> left, Vector128<int> right) { throw null; }
-        public static Vector128<uint> Add(Vector128<uint> left, Vector128<uint> right) { throw null; }
-        public static Vector128<long> Add(Vector128<long> left, Vector128<long> right) { throw null; }
-        public static Vector128<ulong> Add(Vector128<ulong> left, Vector128<ulong> right) { throw null; }
-        public static Vector128<double> Add(Vector128<double> left, Vector128<double> right) { throw null; }
+        public static Vector128<T> Add<T>(Vector128<T> left, Vector128<T> right) where T : struct { throw null; }
```

Comment:
nit: If these were merged (and I still **don't** think it is a good idea), most of these would/could become generic.